.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,89 @@
+# Workflow to build the documentation
+name: Documentation builder
+
+on:
+  push:
+    branches:
+      - main
+      # Release branches
+      - "[0-9]+.[0-9]+.X"
+  pull_request:
+    branches:
+      - main
+      - "[0-9]+.[0-9]+.X"
+
+jobs:
+  # Build the documentation against the minimum version of the dependencies
+  doc-min-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout scikit-learn
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+
+      - name: Build documentation
+        run: bash build_tools/github/build_doc.sh
+        env:
+          OMP_NUM_THREADS: 2
+          MKL_NUM_THREADS: 2
+          CONDA_ENV_NAME: testenv
+          PYTHON_VERSION: 3.8
+          NUMPY_VERSION: min
```

Comment:
With https://github.com/scikit-learn/scikit-learn/pull/23429, I suspect this needs to be updated.

Reply:
Updated!